Resources/Prototypes/SS220/Entities/Mobs/NPCs/pets.yml

```
@@ -0,0 +1,34 @@
+- type: entity
+  name: Рокси
+  parent: MobSecDog
+  id: MobSecDogRoxi
+  description: Охраная собака — помощница офицеров, по поимки злодеев. Больно кусается, лучший офицер станции.

```

```suggestion
  description: Не стоит шутить с ней - в списке её наград есть медаль за отвагу.
```
Насколько помню, ни у одного питомца нет описания, которое на 100% причисляло бы их к кому-то. Полагаю, это сделано для того, чтобы их можно было присвоить для кого ты захочешь. Нужно нейтральное описание, как у остальных.